src/biber.ts

```diff
@@ -2,10 +2,26 @@ import * as core from '@actions/core'
 import * as os from 'os'
 import * as tc from '@actions/tool-cache'
 import {BIBER_DL_BASE_PATH, BINARIES, DOWNLOAD} from './constants'
-import {valid} from 'semver'
+import {coerce} from 'semver'
+
+const validBiberVersion = (biberVersion: string) => {
+  const biberSemVer = coerce(biberVersion)
+
+  if (biberSemVer === null) {
+    core.debug(
+      `Invalid biber version: ${biberVersion}. Defaulting to latest version`
+    )
+    return 'current'
+  }
+  if (biberSemVer.patch !== 0) {
+    return biberSemVer.version
+  }
+
+  return `${biberSemVer.major}.${biberSemVer.minor}`
+}
```

Comment:
Could you please add some tests to this newly introduced method?

Reply:
Ok, ty for your feedback. I addressed your suggestions in the latest commit